torch/testing/_internal/common_utils.py

```diff
@@ -1568,8 +1572,18 @@ def check_if_enable(test: unittest.TestCase):
         if not TEST_WITH_SLOW:
             raise unittest.SkipTest("test is slow; run with PYTORCH_TEST_WITH_SLOW to enable test")
     sanitized_test_method_name = remove_device_and_dtype_suffixes(test._testMethodName)
-    if not IS_SANDCASTLE and "DISABLED_TESTS_DICT" in os.environ:
-        disabled_tests_dict = json.loads(os.environ["DISABLED_TESTS_DICT"])
+
+    if not IS_SANDCASTLE:
+        disabled_tests_dict = {}
+
+        if "DISABLED_TESTS_DICT" in os.environ:
+            disabled_tests_dict = json.loads(os.environ["DISABLED_TESTS_DICT"])
+        elif os.path.exists(IMPORT_DISABLED_TESTS):
+            with open(IMPORT_DISABLED_TESTS, 'r') as fp:
+                disabled_tests_dict = json.loads(fp.read())
+        else:
+            print(f'Fail to load {IMPORT_DISABLED_TESTS}, no test will be skipped')
```

Comment:
The for loop below should not run in this case right?

Reply:
Yeah, it fails back to an empty dict, so there won't be anything to loop over

Reply:
Shouldn't we use `warnings.warn` here?